src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -71,79 +72,74 @@ public void storeMessage(Message message) {
    * @return a list of messages posted by all users, or empty list if no user has ever posted a
    *     message. List is sorted by time descending.
    */
-public List<Message> getAllMessages(){
+  public List<Message> getAllMessages() {
 
-  Query query = new Query("Message")
-    .addSort("timestamp", SortDirection.DESCENDING);
-  PreparedQuery results = datastore.prepare(query);
+    Query query = new Query("Message").addSort("timestamp", SortDirection.DESCENDING);
+    PreparedQuery results = datastore.prepare(query);
 
-  return convertEntitiesToMessages(results);
- }
+    return convertEntitiesToMessages(results);
+  }
 
```

Comment:
@acolwell I'm not sure why these whitespace changes are happening, is it due to the google plugin fixing things, or am I creating whitespace changes that I need to revert?

Reply:
yes. I haven't had a chance to fix all the existing style violations because I was waiting for Alan's pull request to get merged. I'm file with just fixing them as part of this change.